plugins/CoreHome/angularjs/common/directives/focus-anywhere-but-here.js

```diff
@@ -22,54 +22,8 @@
             restrict: 'A',
             link: function(scope, element, attr, ctrl) {
 
-                var isMouseDown = false;
```

Comment:
Any reason why this directive wasn't fully removed like the others?

Reply:
No, I think I just forgot to review this one myself before pinging, apologies.